### ISSUE TITLE
Odoo 12.0

### ISSUE THREAD

**Issue by vbezl** [hootel/hootel#80]
Hi everyone,

Do you have plans to migrate to v12.0? How much time would it require on your opinion?

I've tried to launch it on Odoo 12 - after some issues with installation (mostly res.config.settings) - JS console shows error: 
View type 'pms' is not present in the view registry.

and Odoo shows error:
Error:
Uncaught TypeError: Cannot read property 'type' of undefined

http://localhost:8069/web/static/src/js/chrome/action_manager_act_window.js:334
Traceback:
TypeError: Cannot read property 'type' of undefined
    at Array.<anonymous> (http://localhost:8069/web/static/src/js/chrome/action_manager_act_window.js:334:72)
    at Array.<anonymous> (http://localhost:8069/web/static/lib/jquery/jquery.js:3276:89)
    at fire (http://localhost:8069/web/static/lib/jquery/jquery.js:3119:58)
    at Object.add [as done] (http://localhost:8069/web/static/lib/jquery/jquery.js:3165:49)
    at Array.<anonymous> (http://localhost:8069/web/static/lib/jquery/jquery.js:3275:77)
    at Function.each (http://localhost:8069/web/static/lib/jquery/jquery.js:383:58)
    at Object.<anonymous> (http://localhost:8069/web/static/lib/jquery/jquery.js:3272:56)
    at Function.Deferred (http://localhost:8069/web/static/lib/jquery/jquery.js:3332:30)
    at Object.then (http://localhost:8069/web/static/lib/jquery/jquery.js:3271:55)
    at Object.<anonymous> (http://localhost:8069/web/static/src/js/chrome/action_manager_act_window.js:332:32)

**Comment by DarioLodeiros** [comment]
Hi! 

We hace plans to migrate to v12, of course, but the work in v11 is not
finished yet. Once this is finished, we will study new functionalities
to be developed and the possibility of migrating it to 12, depending on
the priorities we will do before, one or another thing. You must know
that this project is WIP, we need at least 2 months to stabilize it in
v11. 

Thanks!

**Comment by vbezl** [comment]
Thanks for your answer. Looking forward to see it in v12.
I'm using some JS parts of your calendar module in my own project on v11, but was researching migration to v12 and understood that they have changed a lot in JS framework, so it has to be rewritten for v12... staying on v11 for now :)
